src/components/CryptoCards.jsx

```diff
@@ -1,14 +1,44 @@
-import React, {useState, useEffect} from 'react'
+import React, { useState, useEffect } from 'react'
+import { getCoinData } from '../modules/dataCenter'
+import { Card, Grid } from 'semantic-ui-react';
 
 const CryptoCards = () => {
+  const [currencies, setCurrencies] = useState([])
+  const [errorMessage, setErrorMessage] = useState()
 
-  
+  useEffect(() => {
+    setTimeout(async () => {
+      try {
+        let response = await getCoinData()
+        setCurrencies(response.currencies)
+      }
+      catch (error) {
+        setErrorMessage("Couldn't render information. Please update.")
+      }
+    }, 1500)
```

Comment:
Why are you setting a time out here and not fetching the coins directly on render?

Reply:
I wrote it up in the PR 😏
But it's because it will throw a 500 if the second request is sent too fast (because of the limit)

Reply:
okey cool